integration_tests/test_basic.py

```
@@ -47,7 +47,7 @@ def test_simple(mantra, connect_mantra, tmp_path, check_reserve=True):
         account = cli.account(addr)["account"]
         assert account["type"] == "/cosmos.vesting.v1beta1.DelayedVestingAccount"
         assert account["value"]["base_vesting_account"]["original_vesting"] == [
```

Ruff (0.12.2)

Use of `assert` detected

(S101)